css/inner-page.css

```diff
@@ -71,6 +60,8 @@ h3[id] {
     font-size: 24px;
     line-height: 32px;
     margin-bottom: var(--space-unit);
+    /* todo review use of hyphens */
```

Comment:
Thanks! Please use our standard syntax to make sure we can search for it case-sensitively.

Reply:
done